e2e/playwright/tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+    "compilerOptions": {
+        "esModuleInterop": true,
+        "strict": true,
+        "strictNullChecks": true,
+        "skipLibCheck": true,
+        "noEmit": true,
+        "baseUrl": ".",
+        "paths": {
+            "@mattermost/client/*": [
+                "../../../mattermost/webapp/platform/client/lib/*"
+            ],
+            "@mattermost/types/*": [
+                "../../../mattermost/webapp/platform/types/lib/*"
+            ],
+            "@e2e-support/*": [
+                "../../../mattermost/e2e-tests/playwright/support/*"
+            ],
+            "@e2e-test.config": [
+                "../../../mattermost/e2e-tests/playwright/test.config.ts"
+            ],
+            "@e2e-test.playwright-config": [
+                "../../../mattermost/e2e-tests/playwright/playwright.config.ts"
+            ],
+            "@playwright/test": [
+                "../../../mattermost/e2e-tests/playwright/node_modules/@playwright/test"
+            ],
+            "@e2e-types": ["../../../mattermost/e2e-tests/playwright/types.ts"]
+        }
+    },
+    "include": ["./**/*"]
```

Comment:
This `include` may need to be tweaked as we expand and share code among the different plugin projects